pom.xml

```diff
@@ -76,7 +76,7 @@
     <jcr.version>2.0</jcr.version>
     <maven-surefire-plugin.reuseForks>false</maven-surefire-plugin.reuseForks>
     <ehcache-core.version>2.5.1</ehcache-core.version>
-    <jackrabbit-core.version>2.14.2</jackrabbit-core.version>
+    <jackrabbit-core.version>2.16.5</jackrabbit-core.version>
```

Comment:
Please move the version property to the parent pom repository and use dependency management.

Reply:
Done.